src/modules/peek/Peek.FilePreviewer/FilePreview.xaml.cs

```diff
@@ -21,6 +22,8 @@
 using Peek.FilePreviewer.Previewers.Interfaces;
 using Peek.UI.Telemetry.Events;
 
+using Logger = Peek.Common.Helpers.Logger;
```

Comment:
Why are we adding this line?

Reply:
Build gave error with ManagedCommon library